campaigns/serializers.py

```
@@ -0,0 +1,199 @@
+        ]
+
+    owner = AccountSerializer()
+    recipient = AccountSerializer()
+    token = TokenSerializer()
+    status = serializers.SerializerMethodField()
+
+    def get_status(self, obj):
+        """
+        Get campaign status: active, ended, or upcoming
+        """
+        from django.utils import timezone
```

Ruff (0.14.5)

Redefinition of unused `timezone` from line 1

Remove definition: `timezone`

(F811)